lightning/src/routing/scoring.rs

```diff
@@ -2119,6 +2119,29 @@ mod tests {
 		assert_eq!(deserialized_scorer.channel_penalty_msat(42, 500, 1_000, &source, &target), 371);
 	}
 
+	#[test]
+	fn scores_realistic_payments() {
+		// Shows the scores of "realistic" sends of 100k sats over channels of 1-10m sats (with a
+		// 50k sat reserve).
```

Comment:
Do we take into account the reserve? Maybe for first hops only?

Reply:
What do you mean? The capacity amounts are all 50k sats short of the round number. We probably don't need to care, I just did that case I was seeing strange scores for some channels and copied the amounts over to the test while exploring.

Reply:
As in, the capacity passed to the scorer by `get_route` doesn't deduct the reserve, IIRC.